src/logger/exception-log/exception-log.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { ExceptionLogEntity } from '../entities/exception-log.entity';
+
+@Injectable()
+export class ExceptionLogService {
+  constructor(
+    @InjectRepository(ExceptionLogEntity)
```

Comment:
This line is not necessary, remove it. you have to use the repository, take a look at UserRepository we create at the class

Reply:
Ok, done.